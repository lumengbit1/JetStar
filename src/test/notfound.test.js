import React from 'react';
import 'jest-styled-components';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import AppNotFound from '../components/notfound';

describe('AppNotFound Test Cases', () => {
  it('1: Component shoule be rendered correctly', async () => {
    const { container } = render(
      <AppNotFound />,
    );

    expect(container).toMatchSnapshot();
  });
});
