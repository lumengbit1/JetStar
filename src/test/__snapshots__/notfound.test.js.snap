// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage Test Cases 1: Component shoule be rendered correctly 1`] = `
.c0 {
  margin: 3.125rem auto;
  max-width: 62.5rem;
}

.c0 h1,
.c0 h2,
.c0 h3,
.c0 h4,
.c0 h5 {
  color: #999;
}

<div>
  <div
    class="c0"
  >
    <h1>
      Sorry, the page you requested is not available
    </h1>
  </div>
</div>
`;
